src/process_utility.c

```diff
@@ -2780,15 +2780,44 @@ process_altercontinuousagg_set_with(ContinuousAgg *cagg, Oid view_relid, const L
 	}
 }
 
+/* Run an alter table command on a relation */
+static void
+alter_table_by_relation(RangeVar *relation, AlterTableCmd *cmd)
+{
+	const Oid relid = RangeVarGetRelid(relation, NoLock, true);
+	AlterTableInternal(relid, list_make1(cmd), false);
+}
+
+/* Run an alter table command on a relation given by name */
+static void
+alter_table_by_name(Name schema_name, Name table_name, AlterTableCmd *cmd)
+{
+	alter_table_by_relation(makeRangeVar(NameStr(*schema_name), NameStr(*table_name), -1), cmd);
+}
+
+/* Alter a hypertable and do some extra processing */
+static void
+alter_table_by_id(int32 hypertable_id, AlterTableStmt *stmt, AlterTableCmd *cmd,
```

Comment:
Nit, for clarity:

```suggestion
alter_table_by_hypertable_id(int32 hypertable_id, AlterTableStmt *stmt, AlterTableCmd *cmd,
```

or


```suggestion
alter_hypertable_by_id(int32 hypertable_id, AlterTableStmt *stmt, AlterTableCmd *cmd,
```